src/main/scala/com/memsql/streamliner/examples/Transformers.scala

```diff
@@ -93,3 +90,35 @@ class JSONCheckIdTransformer extends SimpleByteArrayTransformer {
     sqlContext.createDataFrame(transformedRDD, schema)
   }
 }
+
+class TwitterHashtagTransformer extends SimpleByteArrayTransformer {
+  override def transform(sqlContext: SQLContext, rdd: RDD[Array[Byte]], config: UserTransformConfig, logger: PhaseLogger): DataFrame = {
+    val columnName = config.getConfigString("column_name").getOrElse("hashtags")
+
+    // convert each input element to a JsonValue
+    val jsonRDD = rdd.map(r => byteUtils.bytesToUTF8String(r))
+
+    val hashtagsRDD: RDD[String] = jsonRDD.mapPartitions(r => {
+      // register jackson mapper (this needs to be executed on each partition)
```

Comment:
I'd either elaborate on this comment (i.e. we instantiate this on each partition so that we don't have to instantiate it for every row) or just remove it entirely, since it's not particularly helpful.